### PR TITLE
Move to latest team lints, require Dart 2.19

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: lib/src/vector_math/error_helpers.dart [hunk 1]
```diff
@@ -12,8 +12,8 @@ double relativeError(dynamic calculated, dynamic correct) {
     final diff = (calculated - correct).abs().toDouble();
     return diff / correct;
   }
-  // ignore: return_of_invalid_type
-  return calculated.relativeError(correct);
+  // ignore: avoid_dynamic_calls
+  return calculated.relativeError(correct) as double;
```
Comment (natebosch) [inline lib/src/vector_math/error_helpers.dart:16]: Is there a perf implication on dart2js here?

cc @rakudrama

Comment (kevmoo) [reply lib/src/vector_math/error_helpers.dart:16]: OH! Good point...

Comment (rakudrama) [reply lib/src/vector_math/error_helpers.dart:16]: Why do you need `as double`?
I would expect there is an implicit downcast from `dynamic` .

Comment (kevmoo) [reply lib/src/vector_math/error_helpers.dart:16]: We have a lint to eliminate implicit downcasts

Comment (kevmoo) [reply lib/src/vector_math/error_helpers.dart:16]: I can just revert this line. Not an issue.

**Review by natebosch — APPROVED** [review]

**Pull request merged**